app/models/passenger.rb

```diff
@@ -0,0 +1,16 @@
+class Passenger < ApplicationRecord
+  has_many :trips
+
+  validates :name, presence: true
+  validates :phone_num, presence: true
+
+  def total_charged
+    charged = 0
+    self.trips.each do |trip|
+      charge = trip.convert_money(trip.cost)
+      charged += charge 
+    end
+    return charged
+  end
+
```

Comment:
It's good that you have some business logic here.  You should also include a method to generate a trip for the current passenger assigning a driver.